pkg/cnservice/server.go

```diff
@@ -257,6 +258,9 @@ func (s *service) createMOServer(inputCtx context.Context, pu *config.ParameterU
 	if err := metric.InitSchema(moServerCtx, ieFactory); err != nil {
```

Comment:
Is this type of initialization really feasible.

Reply:
I verified that it was OK